src/storage/accessors/InMemoryDataAccessor.ts

```diff
@@ -23,14 +23,15 @@ type CacheEntry = DataEntry | ContainerEntry;
 
 export class InMemoryDataAccessor implements DataAccessor {
   private readonly base: string;
-  private readonly store: ContainerEntry;
+  // A dummy container with one entry which corresponds to the base
+  private readonly store: { entries: { ''?: ContainerEntry } };
 
   public constructor(base: string) {
     this.base = ensureTrailingSlash(base);
 
     const metadata = new RepresentationMetadata({ path: this.base });
     metadata.addQuads(generateResourceQuads(DataFactory.namedNode(this.base), true));
-    this.store = { entries: {}, metadata };
+    this.store = { entries: { '': { entries: {}, metadata }}};
```

Comment:
For readability, would perhaps suggest

```suggestion
    const rootContainer = { entries: {}, metadata };
    this.store = { entries: { '': rootContainer }};
```

Slightly weird that the root container is not named `/`.

Reply:
I chose `''` to be consistent with all other entries since the baseUrl including the trailing slash get stripped off. But in this case it actually doesn't matter what the string is so I can just change it to `/`

Reply:
Wouldn’t change it for this case; it’s fine as it is then.